contrib/buildinfo/test/src/mill/contrib/BuildInfoTests.scala

```diff
@@ -17,30 +17,55 @@ import utest.framework.TestPath
 
 object BuildInfoTests extends TestSuite {
 
-  val scalaVersionString = "2.12.4"
+  val booleanVal: Boolean = true
+  val byteVal: Byte = 127
+  val shortVal: Short = 32767
+  val intVal: Int = 2147483647
+  val longVal: Long = 9223372036854775807l
+  val floatVal: Float = 3.40282346638528860e+38f
+  val doubleVal: Double = 1.79769313486231570e+308d
+  val charVal: Char = 'Æ'
+  val stringVal: String = "2.12.4"
+
+  val packageName = "foo"
+  val objectName = "bar"
   trait BuildInfoModule extends TestUtil.BaseModule with scalalib.ScalaModule with BuildInfo {
     def millSourcePath =  TestUtil.getSrcPathBase() / millOuterCtx.enclosing.split('.')
-    def scalaVersion = scalaVersionString
+    def scalaVersion = "2.12.6"
```

Comment:
Why is this hard-coded?

Reply:
it's the build for the test and needs a scalaversion.